server/helpers/findLendingHistory.js

```diff
@@ -0,0 +1,29 @@
+import jwt from 'jsonwebtoken';
+import models from '../database/models';
+import responseMessage from './responseMessage';
+
+const { LendingHistory } = models;
+
+const findLendingHistory = async (id, bookId, response) => {
+  try {
+    let bookDuration = '';
+    const lentBook = await LendingHistory.findOne({ where: { userId: id, bookId, type: 'borrowed' }, order: [['createdAt', 'DESC']], limit: 1 });
```

Comment:
The lentBook variable you named this is quite confusing, shouldn't it be lendBook, since you're getting a book from LendingHistory database, looks like you made changes on every single file.

Reply:
The reason I used the word lent is that the book has been lent, would rename it to borrowedBook for better clarification.

Regarding the file changes.
> Borrowing a book affected lendingHistory, users and all their respective locations.
> Fine and Followers were removed because they were not needed again and it affected all their respective locations.
> Favourites had two different spellings in the code base and that was renamed for consistency and affected all their respective locations.
> Author fullname which I earlier created was renamed to authorName because of the fullname was ambiguous which affected all its locations.
> This is the reason for many file changes.